assets/scss/base/_lists.scss

```diff
@@ -58,11 +48,81 @@ ul, nav ul {
 }
 
 .list--inline {
-	li {
-		display: inline;
+  li {
+    display: inline;
+  }
+}
+
+/*
+Numbered List
+
+Defining sections in markup with numbers associated against them using `.list--numbered` on the container and `.numbered` on the element to identify its position
+
+Markup:
+<fieldset class="list--numbered">
+  <legend>Section 1</legend>
+  <div class="form-field-group numbered">
```

Comment:
I don't think we should be putting anything in these lists. the purpose of the examples should be to demonstrate what the list looks like (which should really be an `<ol>` in this case). 

if the code is content agnostic then the example should be too. if the code isn't then it should probably be it's own component.